### PR TITLE
Adding São Paulo data

### PR DESCRIPTION
**Pull request by feromes**

### CODE REVIEW

**Comment by iandees** [comment]
Hello! Please just stick to one pull request. You can keep modifying the existing branch and it will update this pull request. No need to close and re-open.

File: sources/br/sp/sao-paulo-city.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    "lat": "latitude",
+                    "postcode": "cep",
+                    "id": "sqlc",
+                    "addrtype": "uso"
+                },
+                "compression": "zip"
+            }
+        ]
+    }
+}
```
Comment (iandees) [inline sources/br/sp/sao-paulo-city.json:43]: Can you add a newline at the end of the file please?

File: sources/br/sp/sao-paulo-city.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                    "number": "numero",
+                    "street": "logradouro",
+                    "unit": "complemento",
+                    "lon": "longitude",
+                    "lat": "latitude",
+                    "postcode": "cep",
+                    "id": "sqlc",
+                    "addrtype": "uso"
+                },
+                "compression": "zip"
```
Comment (iandees) [inline sources/br/sp/sao-paulo-city.json:39]: Please move the compression line above the conform line.

**Review by iandees — CHANGES_REQUESTED** [review]
Thanks for this addition.

Are you able to find any license information to link to? Also, can you include some description of what GeoSampa is?

File: sources/br/sp/sao-paulo-city.json (hunk 1 above)
Comment (feromes) [reply sources/br/sp/sao-paulo-city.json]: Done

File: sources/br/sp/sao-paulo-city.json (hunk 2 above)
Comment (feromes) [reply sources/br/sp/sao-paulo-city.json]: Done

**Comment by feromes** [comment]
> Thanks for this addition.
> 
> Are you able to find any license information to link to? Also, can you include some description of what GeoSampa is?

Where can I put description information about GeoSampa? I have tried to put it inside GeoSampa address as a description and the test complained

**Comment by iandees** [comment]
> Where can I put description information about GeoSampa?

You can put it in the description of the pull request if you want.

**Comment by feromes** [comment]
> > Where can I put description information about GeoSampa?
> 
> You can put it in the description of the pull request if you want.

OK, I have put it into note tag inside Json file. Sorry I did it before read your message, but GeoSampa is the official GeoPortal of the municipality of São Paulo city, maintained by the Coordination of Production and Information Analysis (GeoInfo) of the Municipal Department of Urbanism and Licensing

**Comment by iandees** [comment]
Got it, thanks!

**Review by iandees — APPROVED** [review]
Looks good. I'm waiting for the data-pls sample map.

**Comment by feromes** [comment]
Hi @iandees,

Do you have any idea why is it taking so long to get processed?

Thanks,

**Comment by iandees** [comment]
@ingalls it looks like data-pls isn't linking to the right build in the PR here. It looks to maybe be this job in batch? https://batch.openaddresses.io/job/73128

**Comment by ingalls** [comment]
@iandees Looking into this. 

@feromes The reason why it isn't processing quickly is that currently it is running on the same AWS Batch instance as our weekly builds in order to save significant $. In my last deploy yesterday I created the inifial infra to run a second job queue running on the same instances with a higher priority which will let CI jobs skip the queue once there is capacity. I haven't finished wiring this up yet.

**Comment by arch0345** [comment]
Superseded by #6949

**Pull request closed without merging**